### PR TITLE
Update conda package and compass to better support MPI

### LINKED ISSUE #305: Addition of ipython to compass environment
@xylar, happy too add a PR to include `ipython` in the compass environment: https://github.com/MPAS-Dev/MPAS-Tools/blob/master/compass/meta.yaml#L32

This will help with development.  May also want to consider including juypter too to get broader use.

### PR DESCRIPTION
**Pull request by xylar**
This merge makes two scripts into stubs so that the functionality from the scripts is fully available through functions within the `mpas_tools` package:
* The functionality of the paraview VTK extractor is now in `mpas_tools.viz.paraview_extractor`
* Creation of SCRIP files from MPAS meshes is now in `mpas_tools.scrip.from_mpas`
The scripts for this functionality still exist but are just stubs that call the functions.

These changes allow VTK and SCRIP files to be created directly within python scripts, rather than through awkward `subprocess` calls that are causing trouble for conda environments with MPI support.

This merge also updates to the `compass` metapckage to 0.1.3:
 * updates `mpas_tools` to 0.1.10 (not yet released)
 * adds `rasterio`
 * adds `affine`
 * adds `ipython`
 * adds `jupyter`

closes #305

### CODE REVIEW

**Comment by xylar** [comment]
## Testing

I test-build the packages using these steps for building the packages and installing this version of `mpas_tools` and `compass` in a test environment:
```
source ~/miniconda3/etc/profile.d/conda.sh
conda activate base
# if needed:
conda install -y conda-build
conda update -y conda conda-build
rm -rf ~/miniconda3/conda-bld/
conda build --python=3.7 conda_package/recipe/
conda build compass/
conda create -y -n test_compass --use-local "compass=0.1.3=mpi_mpich*"
conda activate test_compass
```

Then, I ran the `QU240wISC` COMPASS test case from https://github.com/MPAS-Dev/MPAS-Model/pull/480

**Comment by xylar** [comment]
The `rasterio` and `affine` packages are needed by https://github.com/MPAS-Dev/MPAS-Model/pull/473, the work for making meshes with a signed-distance function.

File: compass/meta.yaml [hunk 1]
```diff
@@ -45,6 +45,8 @@ requirements:
     - esmf * {{ mpi_prefix }}_*
     - nco
     - pyremap >=0.0.5,<0.1.0
+    - rasterio
+    - affine
```
Comment (pwolfram) [inline compass/meta.yaml]: Just a friendly reminder to include `ipython` and `jupyter` following from #305.  Thanks @xylar

Comment (xylar) [reply compass/meta.yaml]: done

**Comment by xylar** [comment]
@mark-petersen, I think considerably more testing of this COMPASS with MPI is needed before this is ready to merge.  I have been waiting for us to get spun-up initial conditions for the EC60to30 configurations (and the changes related to those) before I start doing testing again.  I should be able to start on that later today or tomorrow.

**Comment by xylar** [comment]
This version of the `mpas_tools` package and `compass` environment have been tested thoroughly as part of MPAS-Dev/MPAS-Model#480.  Merging and releasing...

**Pull request merged**